client/my-sites/site-settings/form-general.jsx

```diff
@@ -348,7 +348,8 @@ export class SiteSettingsFormGeneral extends Component {
 
 	Timezone() {
 		const { fields, isRequestingSettings, translate } = this.props;
-		const guessedTimezone = moment.tz.guess();
+		const guessedTimezone = 'Asia/Calcutta';
```

Comment:
If this can be changed back to `const guessedTimezone = moment.tz.guess();` it would be easier to test other locations having this problem such as Tel Aviv and Canberra by changing the OS timezone.